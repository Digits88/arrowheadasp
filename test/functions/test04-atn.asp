<!-- Test ATN function.
	Returns the arc-tangent of a number.
-->

ATN(10) = <%=ATN(10)%><br>
ATN(0.3) = <%=ATN(0.3)%><br>
ATN("0.3") = <%=ATN("0.3")%><br>
