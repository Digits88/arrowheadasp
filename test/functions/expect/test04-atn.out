<!-- Test ATN function.
	Returns the arc-tangent of a number.
-->

ATN(10) = 1.4711^[0-9]*$<br>
ATN(0.3) = 0.2914^[0-9]*$<br>
ATN("0.3") = 0.2914^[0-9]*$<br>
